<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>929</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item row="0" column="0">
     <widget class="QMdiArea" name="mdiArea"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>929</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="ACreate"/>
    <addaction name="separator"/>
    <addaction name="AOpen"/>
    <addaction name="ASave"/>
    <addaction name="ASave2"/>
    <addaction name="ASaveAll"/>
    <addaction name="separator"/>
    <addaction name="AClose"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Правка</string>
    </property>
    <addaction name="ACut"/>
    <addaction name="ACopy"/>
    <addaction name="APaste"/>
    <addaction name="ADelete"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Вид</string>
    </property>
    <addaction name="AProperties"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>Меню</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="ACreate"/>
   <addaction name="AOpen"/>
   <addaction name="ASave"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockProperties">
   <property name="windowTitle">
    <string>Свойства</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>3</number>
     </property>
     <property name="topMargin">
      <number>3</number>
     </property>
     <property name="rightMargin">
      <number>3</number>
     </property>
     <property name="bottomMargin">
      <number>3</number>
     </property>
     <item row="0" column="0">
      <widget class="QTableView" name="tableProperty">
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>20</number>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>20</number>
       </attribute>
       <attribute name="verticalHeaderHighlightSections">
        <bool>true</bool>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Объекты</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="AAddAnPar"/>
   <addaction name="AAddText"/>
   <addaction name="AAddButton"/>
   <addaction name="AAddIm"/>
   <addaction name="actionTEST"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QGridLayout" name="gridLayout_3">
     <property name="leftMargin">
      <number>3</number>
     </property>
     <property name="topMargin">
      <number>3</number>
     </property>
     <property name="rightMargin">
      <number>3</number>
     </property>
     <property name="bottomMargin">
      <number>3</number>
     </property>
     <item row="0" column="1">
      <widget class="QTableView" name="tableProperty2">
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>20</number>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>20</number>
       </attribute>
      </widget>
     </item>
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="btnAddProp">
         <property name="text">
          <string>Добавить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnDelProp">
         <property name="text">
          <string>Удалить</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="ACreate">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/ico/Folder-Add.ico</normaloff>:/icon/ico/Folder-Add.ico</iconset>
   </property>
   <property name="text">
    <string>Создать</string>
   </property>
  </action>
  <action name="AOpen">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/ico/Folder-Accept.ico</normaloff>:/icon/ico/Folder-Accept.ico</iconset>
   </property>
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="ASave">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/ico/save256.ico</normaloff>:/icon/ico/save256.ico</iconset>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="ASave2">
   <property name="text">
    <string>Сохранить как ...</string>
   </property>
  </action>
  <action name="ASaveAll">
   <property name="text">
    <string>Сохранить всё</string>
   </property>
  </action>
  <action name="AClose">
   <property name="text">
    <string>Закрыть</string>
   </property>
  </action>
  <action name="AProperties">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/ico/options.ico</normaloff>:/icon/ico/options.ico</iconset>
   </property>
   <property name="text">
    <string>Свойства</string>
   </property>
  </action>
  <action name="AAddAnPar">
   <property name="text">
    <string>Ан. параметр</string>
   </property>
   <property name="toolTip">
    <string>Ан. параметр</string>
   </property>
  </action>
  <action name="ACut">
   <property name="text">
    <string>Вырезать</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="ACopy">
   <property name="text">
    <string>Копировать</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="APaste">
   <property name="text">
    <string>Вставить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="ADelete">
   <property name="text">
    <string>Удалить</string>
   </property>
  </action>
  <action name="actionTEST">
   <property name="text">
    <string>TEST</string>
   </property>
   <property name="toolTip">
    <string>test</string>
   </property>
  </action>
  <action name="AAddText">
   <property name="text">
    <string>Текст</string>
   </property>
   <property name="toolTip">
    <string>Текст</string>
   </property>
  </action>
  <action name="AAddButton">
   <property name="text">
    <string>Кнопка</string>
   </property>
  </action>
  <action name="AAddIm">
   <property name="text">
    <string>Им</string>
   </property>
   <property name="toolTip">
    <string>Им</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>AProperties</sender>
   <signal>triggered(bool)</signal>
   <receiver>dockProperties</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>49</x>
     <y>322</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dockProperties</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>AProperties</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>322</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
